src/components/Navbar.js

```diff
@@ -3,13 +3,19 @@ import { AppBar, Toolbar, Typography, Button } from '@mui/material';
 import { Link } from 'react-router-dom';
 import Logout from './Logout';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import Logo from '../images/Logo.png';
 
 const Navbar = () => {
   const { status } = useAuth();
   const isLoggedIn = status === AuthStatus.LoggedIn;
   return (
-    <AppBar position="static">
+    <AppBar position="static" style={{ marginBottom: '2rem' }}>
       <Toolbar>
+        <img
+          src={Logo}
+          alt="Logo"
+          style={{ height: '2.5rem', marginRight: '0.625rem' }}
+        />
         <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
           Music
```

Comment:
Let's rename this to `BeatBazaar`

Reply:
Done, but now one test is failed with error in this line:
`const linkElement = await screen.findByText(/Music/i);
`
I can't merge now.

Reply:
Ah yes this test here was relying on looking for that text in the NavBar: [https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/f92618a9e426845cd5172a76970f62a5654a61cc/src/App.test.js#L20](https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/f92618a9e426845cd5172a76970f62a5654a61cc/src/App.test.js#L20)

Since we are intentionally changing the text in the navbar, we should also change the text that the test is looking for.